Makefile

```diff
@@ -5,3 +5,5 @@ GO=CGO_ENABLED=0 GO111MODULE=on go
 test:
 	$(GO) test ./... -cover
 	$(GO) vet ./...
+	gofmt -l .
+	[ "`gofmt -l -w .`" = "" ]
```

Comment:
actually I'd prefer that this just be
```
[ "`gofmt -l .`" = "" ] 
```
because then running make test won't format the code for you - I think that should be a conscious developer action and is quite simple for folks to learn how to do that.

Reply:
Humm, what it got from last week's discussion was we did want it to modify the files when devs run `make test`. I do get your point about formatting being an explicit action by the devs. 

@tsconn23 Thoughts?